src/NextGen/DonationForm/resources/types.ts

```diff
@@ -200,14 +227,26 @@ export function isField(node: Node): node is Field {
     return node.nodeType === 'field';
 }
 
+export function fieldExists(field: Field, template: FormDesign): boolean {
+    return template.fields.hasOwnProperty(field.type);
+}
```

Comment:
Couple thoughts:
1. Let's move the `{node}Exists` functions to the utilities (or somewhere not here) as they're not really type functions; they're template functions.
2. Let's rename them to `{node}TemplateExists` to be a bit clearer as to what we're checking.

Reply:
resolved e971c1f1